app/views/photos/_photo.html.erb

```diff
@@ -1,3 +1,4 @@
+<% if policy(@user).view_private_profile_content? %>
```

Comment:
redundant with the added conditional in users/show.hmtl.erb

Reply:
I think the reason I wanted to add this was because I couldn't see private profiles, so I tried to work around it this way. Please let me know if there is a better way. Thank you!